### PR TITLE
added transition/results apearance

### PR DESCRIPTION
**Pull request by CodyDiab**

### CODE REVIEW

**Comment by CodyDiab** [comment]
Just a few changes that i wanted to be included if any one were going to pull from this branch.

File: assets/js/keys.js [hunk 1]
```diff
@@ -123,6 +123,13 @@ function setSearchTerm(event) {
 	//call function to fetch product details
 	getAmazonUrl();
 	getWalmartUrl();
+var display=document.querySelector("#results")
+var headerr=document.querySelector("#header")
```
Comment (melissarpowell) [inline assets/js/keys.js]: At first I thought this was a typo with an extra "r", then I realized this is consistent below. Was this intentional?

File: index.html [hunk 2]
```diff
@@ -30,10 +30,10 @@
 
   <!-- Primary Page Layout
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-  <div class="container">
-    <div class="row header">
-
-      <div class="one-half column" style="margin-top: 15%">
+  <div class="container ">
```
Comment (melissarpowell) [inline index.html]: I noticed a few classes have an extra space before the end quote. Is this going to be an issue later when we are trying to identify classes?

File: index.html [hunk 3]
```diff
@@ -30,10 +30,10 @@
...
+    <div id="header" class="row header" style="padding-top: 30%; padding-bottom: 30%">
+    
+      <div class="one-half column " >
```
Comment (melissarpowell) [inline index.html:36]: Same here

File: index.html [hunk 4]
```diff
@@ -43,7 +43,7 @@ <h2 class="main-header eight-columns title">Paper Locator <img class="logo two c
 
       </div>
     </div>
-    <div class="one-half column" style="margin-top: 15%">
+    <div class="one-half column " >
```
Comment (melissarpowell) [inline index.html:46]: Same here

**Review by melissarpowell — APPROVED** [review]
Everything looks good. Thank you!
I was curious about the extra space after the classes.

File: index.html (hunk 2 above)
Comment (bradleypaul) [reply index.html]: It shouldn't be. CSS is pretty open about that.

**Review by bradleypaul — APPROVED** [review]

**Pull request merged**